composer.json

```diff
@@ -13,25 +13,25 @@
         "iignatov/lightopenid": "~1.0",
         "paypal/adaptivepayments-sdk-php": "*",
         "paypal/merchant-sdk-php": "*",
-        "symfony/class-loader": "2.8.30",
-        "symfony/http-foundation": "2.8.30",
-        "symfony/routing": "2.8.30",
-        "symfony/http-kernel": "2.8.30",
-        "symfony/security": "2.8.30",
-        "symfony/config": "2.8.30",
-        "symfony/stopwatch": "2.8.30",
-        "symfony/dependency-injection": "2.8.30",
-        "symfony/debug": "2.8.30",
-        "symfony/console": "2.8.30",
-        "symfony/translation": "2.8.30",
-        "symfony/finder": "2.8.30",
-        "symfony/dom-crawler": "2.8.30",
-        "symfony/css-selector": "2.8.30",
-        "symfony/process": "2.8.30",
-        "symfony/form": "2.8.30",
-        "symfony/templating": "2.8.30",
-        "symfony/framework-bundle": "2.8.30",
-        "symfony/validator": "2.8.30",
+        "symfony/class-loader": "^2.8",
+        "symfony/http-foundation": "^2.8",
+        "symfony/routing": "^2.8",
+        "symfony/http-kernel": "^2.8",
+        "symfony/security": "^2.8",
+        "symfony/config": "^2.8",
+        "symfony/stopwatch": "^2.8",
+        "symfony/dependency-injection": "^2.8",
+        "symfony/debug": "^2.8",
+        "symfony/console": "^2.8",
+        "symfony/translation": "^2.8",
+        "symfony/finder": "^2.8",
+        "symfony/dom-crawler": "^2.8",
+        "symfony/css-selector": "^2.8",
+        "symfony/process": "^2.8",
+        "symfony/form": "^2.8",
+        "symfony/templating": "^2.8",
+        "symfony/framework-bundle": "^2.8",
+        "symfony/validator": "^2.8",
```

Comment:
@davidbeig great! However, we are missing the updated composer.lock only with the Symfony packages

Reply:
Added!